sdk/tests/conformance2/transform_feedback/transform_feedback.html

```diff
@@ -324,6 +346,80 @@
     requestAnimationFrame(completeTransformFeedbackQueryTest);
 }
 
+function runUnboundDeleteTest() {
+    debug("");
+    debug("Testing deleting buffers attached to an unbound transform feedback object");
+
+    // Theoretically it would be possible to verify the result of performing
+    // transform feedback into a deleted buffer object. The buffer would have to
+    // be latched into a VAO as well as into the transform feedback object. In
+    // order to get the results out of the output buffer, it would be necessary
+    // to run transform feedback again, reading from the buffer bound to the
+    // VAO, and writing into a (non-deleted) buffer object latched into the
+    // transform feedback object. It's not possible to arrange things to be able
+    // to copyBufferSubData from the deleted buffer object into a temporary one
+    // for readback.
+
+    // This would be a lot of code to test an unlikely corner case, so instead,
+    // this test verifies simpler behaviors.
+
+    out_add_buffer = gl.createBuffer();
+    const output_buffer_length = Float32Array.BYTES_PER_ELEMENT * 16;
+    gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, out_add_buffer);
+    gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, output_buffer_length, gl.STATIC_DRAW);
+
+    // Set up the transform feedback object
+    tf = gl.createTransformFeedback();
+    gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tf);
+    gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, out_add_buffer);
+
+    // Unbind transform feedback and delete out_add_buffer.
+    gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, null);
+    gl.deleteBuffer(out_add_buffer);
+    debug("isBuffer should report false after deletion");
+    shouldBe("gl.isBuffer(out_add_buffer)", "false");
+
+    gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tf);
+    debug("Transform feedback object should keep output buffer alive");
+    shouldBe("gl.getIndexedParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING, 0)", "out_add_buffer");
+    // Deleting the buffer again while the transform feedback is bound shouldn't unbind it.
+    gl.deleteBuffer(out_add_buffer);
+    debug("Deleting output buffer again should be a no-op");
+    shouldBe("gl.getIndexedParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING, 0)", "out_add_buffer");
```

Comment:
Maybe test unbinding and rebinding the transform feedback object as well? Just to make sure that unbinding the transform feedback object doesn't somehow propagate to the buffer being unbound.

Reply:
Thank you, good suggestion. Added that test here.